Makefile.build

```diff
@@ -316,11 +316,21 @@ $(LIBCOQRUN): kernel/byterun/coq_jumptbl.h $(BYTERUN)
 	cd $(dir $(LIBCOQRUN)) && \
 	$(OCAMLFIND) ocamlmklib -oc $(COQRUN) $(notdir $(BYTERUN))
 
-kernel/byterun/coq_jumptbl.h : kernel/byterun/coq_instruct.h kernel/byterun/make_jumptbl.sh
-	kernel/byterun/make_jumptbl.sh $< $@
+kernel/genOpcodeFiles.exe: kernel/genOpcodeFiles.ml
+	$(SHOW)'OCAMLC    $<'
+	$(HIDE)cd $(dir $<) && $(OCAMLC) -o $(notdir $@) $(notdir $<)
+
+kernel/byterun/coq_instruct.h: kernel/genOpcodeFiles.exe
+	$(SHOW)'WRITE $@'
+	$(HIDE)(cd kernel/byterun && ../$(shell basename $<) enum)
 
-kernel/copcodes.ml: kernel/byterun/coq_instruct.h kernel/make_opcodes.sh kernel/make-opcodes
-	kernel/make_opcodes.sh $< $@
+kernel/byterun/coq_jumptbl.h: kernel/genOpcodeFiles.exe
+	$(SHOW)'WRITE $@'
+	$(HIDE)(cd kernel/byterun && ../$(shell basename $<) jump)
```

Comment:
I am not make expert but a couple of comments on this rule:
- why do you spawn a subshell with `(...)` ? As far as I know make already does spawn a fresh shell for each line. In fact, this is likely the reason the output status was masked. AFAICT the right thing to do is to remove the `()`
- I'd also prefer if the `genOpcodeFiles` would output to stdout. This makes renaming and moving around files much easier, it should also simplify the code as you don't need to open a file with all the complex error handling than has to be done, and moreover you wouldn't need to use `cd` and `basename` as something like this should work:
` $< jump > $@`